api/src/api/application_alpha/application_route.py

```diff
@@ -0,0 +1,33 @@
+import logging
+
+import src.adapters.db as db
+from src.adapters.db import flask_db
+from src.api import response
+from src.api.application_alpha.application_blueprint import application_blueprint
+from src.api.application_alpha.application_schemas import (
+    ApplicationStartRequestSchema,
+    ApplicationStartResponseSchema,
+)
+from src.auth.api_key_auth import api_key_auth
+from src.services.applications.create_application import create_application
+
+logger = logging.getLogger(__name__)
+
+
+@application_blueprint.post("/application_alpha/start")
```

Comment:
For the path, we want `/applications/...` - the alpha/versioning bit is already added by the blueprint itself.